sentinel-transport/sentinel-transport-simple-http/src/main/java/com/alibaba/csp/sentinel/transport/heartbeat/client/SimpleHttpClient.java

```diff
@@ -184,6 +213,50 @@ private String encodeRequestParams(Map<String, String> paramsMap, Charset charse
         }
     }
 
+	private X509TrustManager getX509TrustManager() {
+		if (x509TrustManager == null) {
+			x509TrustManager = new X509TrustManager() {
+				public boolean isServerTrusted(X509Certificate[] certs) {
+					return true;
+				}
+
+				public boolean isClientTrusted(X509Certificate[] certs) {
+					return true;
+				}
+
+				@Override
+				public void checkServerTrusted(X509Certificate[] certs, String authType)
+						throws CertificateException {
+				}
+
+				@Override
+				public void checkClientTrusted(X509Certificate[] certs, String authType)
+						throws CertificateException {
+				}
+
+				@Override
+				public X509Certificate[] getAcceptedIssuers() {
+					return null;
+				}
+			};
+		}
+		return x509TrustManager;
+	}
+
+	private SSLSocketFactory getSSLSocketFactory() {
```

Comment:
The All-In-One approach makes the class a little ambiguous.
Suggest abstracting the socket creation into different implementations outside.